makeflow/src/makeflow.c

```diff
@@ -1866,7 +1804,9 @@ int main(int argc, char *argv[])
 			case LONG_OPT_SANDBOX:
 				makeflow_hook_register(&makeflow_hook_sandbox);
 				break;
-		}
+			case LONG_OPT_LOOP_DEVICE:
```

Comment:
I am adding this here as the loop device was closely tied in implementation with resource monitor in the node complete function. Loop device now only minor ties with Makeflow and resource monitor, so it can be moved.